tests/introspection/ApiSelectorTest.cs

```diff
@@ -592,6 +592,24 @@ protected virtual bool Skip (Type type, string selectorName)
 					return true;
 				}
 				break;
+			case "MPSImageLaplacianPyramid":
+				switch (selectorName) {
+				case "initWithDevice:kernelWidth:kernelHeight:weights:":
+					return true;
+				}
+				break;
+			case "MPSImageLaplacianPyramidSubtract":
+				switch (selectorName) {
+				case "initWithDevice:kernelWidth:kernelHeight:weights:":
+					return true;
+				}
+				break;
+			case "MPSImageLaplacianPyramidAdd":
```

Comment:
the last two cases could be just under the first one - as long as they remain identical (like they currently are)